<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>900</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Maslova Lab07</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QTableWidget" name="segmentsTable">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>301</width>
        <height>141</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>301</width>
        <height>141</height>
       </size>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>21</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>41</number>
      </attribute>
      <column>
       <property name="text">
        <string>(X1, Y1)</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>(X2, Y2)</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="1" rowspan="8">
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="segmentsLayout">
      <item>
       <layout class="QGridLayout" name="segmentLayout">
        <item row="0" column="1">
         <widget class="QSpinBox" name="x1SB">
          <property name="maximum">
           <number>1164</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="x2SB">
          <property name="maximum">
           <number>1164</number>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QSpinBox" name="y2SB">
          <property name="maximum">
           <number>874</number>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QSpinBox" name="y1SB">
          <property name="maximum">
           <number>874</number>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="y2Lbl">
          <property name="text">
           <string>Y2:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="x1Lbl">
          <property name="text">
           <string>X1:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="y1Lbl">
          <property name="text">
           <string>Y1:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="x2Lbl">
          <property name="text">
           <string>X2:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="addSegmentBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Добавить
отрезок</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QTableWidget" name="selecterTable">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>301</width>
        <height>91</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>301</width>
        <height>91</height>
       </size>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>21</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>41</number>
      </attribute>
      <column>
       <property name="text">
        <string>Xл</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Xп</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Yн</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Yв</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="3" column="0">
     <layout class="QVBoxLayout" name="selectersLayout">
      <item>
       <layout class="QGridLayout" name="selecterLayout">
        <item row="0" column="1">
         <widget class="QSpinBox" name="xLeftSB">
          <property name="maximum">
           <number>1164</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="xRightSB">
          <property name="maximum">
           <number>1164</number>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QSpinBox" name="yLowSB">
          <property name="maximum">
           <number>874</number>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QSpinBox" name="yTopSB">
          <property name="maximum">
           <number>874</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="xLeftLbl">
          <property name="text">
           <string>Xл:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="yTopLbl">
          <property name="text">
           <string>Yв:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="xRightLbl">
          <property name="text">
           <string>Xп:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="yLowLbl">
          <property name="text">
           <string>Yн:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="setSelecterBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Задать
отсекатель</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QGridLayout" name="colorLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="segmentsColorLbl">
        <property name="text">
         <string>Цвет отрезков:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="selecterColorLbl">
        <property name="text">
         <string>Цвет отсекателя:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="segmentsColorBtn">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>62</width>
          <height>33</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>62</width>
          <height>33</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="selecterColorBtn">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>62</width>
          <height>33</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>62</width>
          <height>33</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:0, stop:0 rgba(255, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="resultColorLbl">
        <property name="text">
         <string>Цвет результата:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="resultColorBtn">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>62</width>
          <height>33</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>62</width>
          <height>33</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:0, stop:0 rgba(38, 255, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="selectBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Отсечь</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="clearBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Очистить</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="roolsBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Правила ввода</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
